micrometer-core/src/main/java/io/micronaut/configuration/metrics/binder/netty/InstrumentedEpollEventLoopGroupFactory.java

```diff
@@ -118,21 +118,6 @@ public EventLoopGroup createEventLoopGroup(int threads, Executor executor, @Null
                 instrumentedEventLoopTaskQueueFactory);
     }
 
-    /**
-     * Creates a default EpollEventLoopGroup.
-     *
-     * @param ioRatio The io ratio.
-     * @return A EpollEventLoopGroup.
-     */
-    @Override
-    public EventLoopGroup createEventLoopGroup(@Nullable Integer ioRatio) {
-        return new EpollEventLoopGroup(0, (Executor) null,
-                DefaultEventExecutorChooserFactory.INSTANCE,
-                DefaultSelectStrategyFactory.INSTANCE,
-                RejectedExecutionHandlers.reject(),
-                instrumentedEventLoopTaskQueueFactory);
-    }
-
```

Comment:
The compilation failed because the method was removed in core: https://github.com/micronaut-projects/micronaut-core/commit/8dc33cbb0a27d2875e6771b57f46c0d494aa9509

I'm not 100% sure if the right fix here is to delete these method or not.